src/lib/comms/include/sol-http-server.h

```diff
@@ -56,25 +56,126 @@ extern "C" {
 struct sol_http_server;
 struct sol_http_request;
 
+/**
+ * It creates a http's server on that will bind on all interfaces in the given port.
```

Comment:
HTTP. It's an acronym. No need to start with `It`:

   Creates an HTTP server, binding on all interfaces at the specified @port.